src/components/Movies/MovieForm.tsx

```diff
@@ -23,26 +32,44 @@ const MovieForm: FC<MovieFormProps> = ({
   genres= [],
   onClose,
 }) => {
-  const [inputTitle, setInputTitle] = useState(title)
-  const [inputRating, setInputRating] = useState<number>(rating)
-  const [inputRuntime, setInputRuntime] = useState<number>(runtime)
-  const [inputGenres, setInputGenres] = useState<string[]>(genres)
-  const [inputReleaseDate, setInputReleaseDate] = useState<string>(releaseDate)
+  const [formValues, setFormValues] = useState<MovieFormValues>({
+    title,
+    rating,
+    runtime,
+    genres,
+    releaseDate,
+    overview,
+  })
 
-  const handleTitleChange = (e: { target: { value: React.SetStateAction<string>; }; }) => {
-    setInputTitle(e.target.value)
+  const handleTitleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setFormValues(
+      prevState => ({ ...prevState, title: e.target.value })
```

Comment:
Just add a `name` attribute to each input and you'll be able to have only 1 function for updating values.

`{ ...prevValues, [e.target.name]: e.target.value }`

Reply:
Done